projects/envoy/build.sh

```diff
@@ -119,6 +119,8 @@ do
   mkdir -p "${CORPUS_UNTAR_PATH}"
   tar -C "${CORPUS_UNTAR_PATH}" -xvf bazel-bin/"${t}"_corpus_tar.tar
   TARGET_BASE="$(expr "$t" : '.*/\(.*\)_fuzz_test')"
+  # There may be *.dict files in this folder that need to be moved into the OUT dir.
+  find "${CORPUS_UNTAR_PATH}" -type f -name *.dict -exec mv -n {} "${OUT}"/ \;
```

Comment:
Why is there a `\;` on the end? Also, probably want to do `"*.dict"` to avoid potential shell match on any files in CWD.

Reply:
The exec command ends at a `;` (man page: "All following arguments to find are taken to be arguments to the command until an argument consisting of `;` is encountered.") The example in the man page show the semi-colon escaped, and it seems by trail I have to do that as well.